ingq/book/application/book_service.py

```
@@ -3,17 +3,20 @@
 
 from sqlalchemy.orm import Session
 
+from book.application.image_validation_service import ImageValidationService
```

Pylint (3.3.7)

[error] Unable to import 'book.application.image_validation_service'

(E0401)